src/js_native_api_v8.cc

```diff
@@ -816,12 +816,7 @@ napi_status napi_define_class(napi_env env,
     }
 
     v8::Local<v8::Name> property_name;
-    napi_status status =
-        v8impl::V8NameFromPropertyDescriptor(env, p, &property_name);
-
-    if (status != napi_ok) {
-      return napi_set_last_error(env, status);
```

Comment:
`STATUS_CALL` simply returns on not `napi_ok` without calling `napi_set_last_error`. The behavior seems not equivalent with the change. is is intentional?

Reply:
@legendecas yes, it is, because it is assumed that whichever call returned non-`napi_ok` did so via `return napi_set_last_error(napi_<not_ok>);` I checked to make sure that was the case in all locations where I made the change.